test_scripts/Defects/5_0/1409_SDL_does_not_send_EndService_for_Video_Audio_Service_upon_IGNITION_OFF.lua

```diff
@@ -0,0 +1,107 @@
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1409
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) SPT is registered using v3 protocol and activated on HMI
+-- Description:
+--
+-- Steps to reproduce:
+-- 1) Start Video Service -> SDL send StartServiceACK to mobile
+-- 2) Start Audio service -> SDL send StartServiceACK to mobile
+-- 3) Send IGNITION_OFF from HMI: On HTML5 HMI click Exit Application and select Ignition Off from drop down
+-- Expected result:
+-- SDL sends End Service (Control Frame 0x04) for: RPC, Audio and Video services
+-- Actual result:
+-- SDL sends End Service for RPCservice only.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Defects/commonDefects')
+local events = require('events')
+
+config.defaultProtocolVersion = 3
+
+--[[ Local Variables ]]
+
+--[[ Local Functions ]]
+local function appStartAudioStreaming(self)
+  self.mobileSession1:StartService(10)
+  :Do(function()
+      self.hmiConnection:ExpectRequest("Navigation.StartAudioStream")
+      :Do(function(_, data)
+          self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+          self.mobileSession1:StartStreaming(10,"files/MP3_1140kb.mp3")
+          self.hmiConnection:ExpectNotification("Navigation.OnAudioDataStreaming", { available = true })
+        end)
+    end)
+end
+
+local function appStartVideoStreaming(self)
+  self.mobileSession1:StartService(11)
+  :Do(function()
+      self.hmiConnection:ExpectRequest("Navigation.StartStream")
+      :Do(function(_, data)
+          self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+          self.mobileSession1:StartStreaming(11, "files/MP3_4555kb.mp3")
+          self.hmiConnection:ExpectNotification("Navigation.OnVideoDataStreaming", { available = true })
+        end)
+    end)
+end
+
+local function ExitAllApplications(self)
+  StopSDL()
+  self.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications", { reason = "IGNITION_OFF" })
+  local event = events.Event()
+    event.matches = function(_, data)
+          return data.frameType   == 0 and
+              (data.serviceType == 11 or data.serviceType == 10 or data.serviceType == 7) and
+                 data.sessionId   == self.mobileSession1.sessionId and
+                (data.frameInfo   == 4) --EndService
+                 end
+    self.mobileSession1:ExpectEvent(event, "EndService")
+    :Timeout(60000)
+    :Times(3)
+    :ValidIf(function(_, data)
+      if data.frameInfo == 4  and data.serviceType == 11 then
```

Comment:
for loop for all EndServiceACK's can be updated to 
 if data.frameInfo == 4  and data.serviceType == 11 or data.serviceType == 10 or data.serviceType == 7 then
.....
return true
end
return false, "End Service not received"